packages/vue/src/components/__tests__/alias-control.spec.ts

```diff
@@ -0,0 +1,79 @@
+import AliasControl from '../alias-control.vue';
+import { components } from '../../../global-spec';
+import { render, screen } from '@testing-library/vue';
+
+// mock Math.random so that ids are statically generated
+jest.spyOn(global.Math, 'random').mockReturnValue(0.123456789);
+
+describe('AliasControl Component', () => {
+  describe('Label', () => {
+    it('label should render default classnames', async () => {
+      render(AliasControl, {
+        global: {
+          components,
+        },
+        props: {
+          label: 'Username',
+          name: 'username',
+          placeholder: 'Username',
+        },
+      });
+      const label = await screen.findByText('Username');
+      expect(label).toHaveClass('amplify-label');
+    });
+
+    it('label should have `amplify-visually-hidden` class when labelHidden is true', async () => {
+      render(AliasControl, {
+        global: {
+          components,
+        },
+        props: {
+          label: 'Username',
+          name: 'username',
+          placeholder: 'Username',
+          labelHidden: true,
+        },
+      });
+
+      const label = await screen.findByText('Username');
+      expect(label).toHaveClass('amplify-visually-hidden');
```

Comment:
This part specifically would've caught this regression.

Reply:
Would it have though since we aren't sharing class names, and we renamed the class?